templates/docs/content-guidelines.md

```diff
@@ -0,0 +1,190 @@
+
+## Error messages
+
+Error messages can be more frustrating than other notifications. Support the user when something goes wrong.
+
+- Don’t blame the user (Use ‘incorrect password,’ not, ‘you entered your password incorrectly.’)
+- Explain what went wrong.
+- Use verb-first, short instructions on how to fix the issue or get help.
+- Try to match the title action to the CTA action.
+
+Consider thinking about the specific user and the moment they get this error. Do they have a very technical background and want lots of details? Is there a job to be done and they just need to know how to get there?
+
+You can also use this cheat sheet:
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/7d840944-error-message-cheat-sheet.png" alt="A decision tree explains how to decide on the vocabulary relative to an error."></div>
+
+## Dates and times
+
+To support international and US audiences, shorten dates with dashes. For example, 10-SEPT-23.
+
+To save space, you can use all numerals separated by a forward slash using the UK convention of [day]/[month]/[year].
+
+Use numerals for relative dates. For example, ‘2 hours ago’ or ‘Updated 5 days ago’.
+
+## Numbers
+
+Use numerals in the UI when it can help highlight an action and compare two amounts. Use the UK convention for commas and decimals in numbers. For example, £10.00 rather than £10,00.
+
+- **Example from Landscape:**
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/135209d1-numbers-numberals-example-landscape.png" alt="An example of a dialaog in Landscape where the use of numbers is shown"></div>
+
+- Note: this guidance differs from the Copy Style Guide to spell out numerals less than 10. Follow the Copy Style Guide for copy that is not part of the UI.
+
+## Videos
+
+When you embed a video in a web page, you don’t need a ‘watch’ CTA as well. Use a clear header to explain to the user what they will get from clicking on the video. For example, what they will learn.
+
+## Writing notifications
+
+There are three basic concepts for UX copy that we use: attributes and objects, and actions.
+
+- **Attributes:** The components that make up a piece of content. For example, a title, description.
+- **Objects:** Usually nouns, the things users interact with.
+- **Actions:** Often verbs, things users can do with objects.
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/cde94563-lxd-context.png" alt="A capture of a form of the LXD UI, including the app side navigation. The context relevant for an error message is outlined on the screenshot."></div>
+
+We should be specific about objects and actions and not have duplicate terms. Our messages typically include three attributes:
+
+- **Title**
+- **Description**
+- **CTA**
+
+**Example from Anbox:**
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/e53e5133-reset-dialog.png" alt="Example of a reset dialog in Anbox with contextualized title, description and CTA."></div>
```

Comment:
For images it would be good to add some arbitrary size to img tag for them not to spill too large? `width=720` or something?

We have images in documentation layout page (they are naturally 720px there: https://vanilla-framework-5425.demos.haus/docs/layouts/documentation)